core/src/test/resources/logback-test.xml

```diff
@@ -41,7 +41,8 @@
     </logger>
     -->
 
-    <!-- netty (mock server) produces a huge amount of DEBUG output during tests without this line -->
+    <!-- netty (mock server) produces a huge amount of DEBUG output during tests without these lines -->
+    <logger name="org.mockserver.log" level="WARN" />
```

Comment:
this is totally unrelated to this PR, but I got tired of seeing so many lines of output in unit tests from mockserver, so I quieted them down.